### DISCUSSION TITLE
academy/deploying-your-code/inputs-outputs

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1066] [Comments]
# academy/deploying-your-code/inputs-outputs

Learn to accept input into your actor, do something with it, and then return output. Actors can be written in any language, so this concept is language agnostic.

https://docs.apify.com/academy/deploying-your-code/inputs-outputs

**Comment by rjthompson2** [comment]
dataset.push_items('OUTPUT', value=[json.dumps(data, indent=4)]) threw an error because value wasn't a valid parameter.

Use instead: dataset.push_items([data)])
ensure data is a dictionary or json.

**Reply by jancurn** [reply]
Thanks for the comment. @vdusek pls can you check it?

**Reply by vdusek** [reply]
Hi @rjthompson2, thanks for noticing, the fix is on its way.
